docs/sphinx/source/developer_notes.rst

```diff
@@ -21,15 +21,65 @@ you'll need to clone the RdTools source repository from Github with e.g.
 from the command line, or using a GUI git client like Github Desktop.  This
 will clone the entire git repository onto your computer.  
 
-Running the test suite
-----------------------
+Installing RdTools dependencies
+-------------------------------
 
-RdTools uses `pytest <https://docs.pytest.org/en/latest/>`_ to run its test
-suite.  If you don't already have it installed:
+The packages necessary to run RdTools itself can be installed with ``pip``.
+You can install the dependencies along with RdTools itself from 
+`PyPI <https://pypi.org/project/rdtools/>`_:
 
 ::
 
-    pip install pytest
+    pip install rdtools
+
+This will install the latest official release of RdTools.  If you want to work
+with a development version and you have cloned the Github repository to your
+computer, you can also install RdTools and dependencies by navigating to the
+repository root and running:
+
+::
+
+    pip install .
+
+This will install the development version of RdTools along with the current
```

Comment:
Technically it will install whatever branch you are on, which may not be development